core/src/avm2/globals/flash/events/PermissionEvent.as

```diff
@@ -0,0 +1,36 @@
+package flash.events
+{
+    
+    public final class PermissionEvent extends Event
```

Comment:
Perhaps this class should also be removed, as the `status` property, and all 3 methods (incl. the ctor.) are marked in the docs as AIR-only.